elements/lisk-p2p/src/peer.ts

```diff
@@ -670,7 +657,7 @@ export const connectAndRequest = async (
 				procedure,
 			};
 			// Ideally, we should JSON-serialize the whole NodeInfo object but this cannot be done for compatibility reasons, so instead we put it inside an options property.
-			const clientOptions: ClientOptionsUpdated = {
+			const clientOptions: ClientOptions = {
 				hostname: basicPeerInfo.ipAddress,
```

Comment:
Why don't we use `pingTimeoutDisabled: false, pingTimeout: 20000,` as suggested in the commit 6e0a8543f0b10733489b55951aba407935632443 ?

Reply:
@diego-G `pingTimeoutDisabled: false` is the default, so we don't need to set it explicitly.

Reply:
@diego-G as Marius mentioned, it's safer and more compatible to stick to the defaults in SC.

Reply:
The timeout too? What about adding a simple scenario checking the behaviour?